setup.py

```diff
@@ -24,7 +24,7 @@ def read(*parts):
     url='https://github.com/python-babel/django-babel/',
     packages=find_packages(exclude=('tests',)),
     install_requires=[
-        'django>=1.4,<1.10',
+        'django>=1.4,<1.11',
         'babel>=1.3',
     ],
     classifiers=[
```

Comment:
> I've used this sort of setup elsewhere for Django projects and it seems to work okay.

You should probably go back and fix those too ;)

Although this isn't a django project, it's a django app so it might be different.